project_event/wizard/project_event_wizard.py

```diff
@@ -0,0 +1,220 @@
+                    'resource_id': task.resource_id.id,
+                    'department_id': task.department_id.id,
+                    'duration': task.duration,
+                    'start_time': task.start_time,
+                    'notes': task.notes,
+                }
+                self.env['task.template.line.wiz'].create(task_vals)
+
+        return {
+            'name': _('Tasks Wizard'),
+            'type': 'ir.actions.act_window',
+            'view_type': 'form',
+            'view_mode': 'form',
+            'res_model': self._name,
+            'views': [(view.id, 'form')],
+            'target': 'new',
+            'res_id': self.id,
+            'context': self.env.context,
+        }
+
+    @api.multi
+    def create_event_from_template(self):
+        self.ensure_one()
+        event_vals = {
+            'name': self.name,
+            'responsible_id': self.event_resp_id.id,
+            'notes': self.event_notes,
+            'project_type': 'event',
+        }
+        event = self.env['project.project'].create(event_vals)
+        for act in self.activity_line_ids:
+            tasks = self.env['task.template.line.wiz'].search([
+                ('event_wiz_id', '=', self.id),
+                ('activity_wiz_id', '=', act.id),
+            ])
+            activity_vals = {
+                'name': act.name,
+                'project_id': event.id,
+                'responsible_id': act.activity_resp_id.id,
+                'activity_category_id': act.activity_category_id.id,
+                'activity_task_type': 'activity',
+                # add room_id
+                'notes': act.notes,
+                'child_ids': [
+                    (0,
+                     0,
+                     {
+                         'name': task.task_name,
+                         'activity_task_type': 'task',
+                         'task_responsible_id': task.task_resp_id.id,
+                         'task_category_id': task.task_category_id.id,
+                         'department_id': task.department_id.id,
+                         'notes': task.notes,
+                         # add resource_id
+                     }
+                     ) for task in tasks],
+            }
+            activity = self.env['project.task'].create(activity_vals)
+
+    @api.onchange('template_id')
+    def _onchange_template_id(self):
+        template = self.env['event.template'].browse(self.template_id.id)
+        self.name = template.name
+        self.event_resp_id = template.temp_resp_id
+        self.event_notes = template.notes
+
+
+class ActivityTemplateLineWiz(models.TransientModel):
+    _name = "activity.template.line.wiz"
+    _description = "Activity Template Line Wizard"
+
+    event_wiz_id = fields.Many2one(
+        'project.event.wizard',
+        string='Activity Template',
+    )
+    template_id = fields.Many2one(
+        'activity.template',
+        string='Activity Template',
```

Comment:
@rimbendhaou : event_wiz_id and template_id have same string Name.  Please keep relevant and meaningful string name for each field declaration (Ignore if not applicable)